quick-install.md

```diff
@@ -0,0 +1,26 @@
+# Install Thunder
+
+### System requirements
+
+You have to install `PHP`, `composer` and `git` on your computer before you can install Thunder. The `composer` requires `git` command for proper functioning.
+
+To install `PHP` please take a look at official [Installation and Configuration](https://www.php.net/manual/install.php) for `PHP`.
+On top of `PHP`, you need to install the required libraries. Thunder installation requires the following `PHP` libraries: `xmlwriter`, `gd`, `mbstring`, `curl`, `json` and `pdo_sqlite`. The installation of `PHP` extensions can defer between operating systems, that's why you should check for detailed instructions on `PHP` [Installation and Configuration](https://www.php.net/manual/install.php).
+
+To install `composer`, you can check the `composer` [installation instructions](https://getcomposer.org/download) and for `git` you can find installation instructions [here](https://git-scm.com/downloads).
+
+### Quick start installation
+
+To start quickly, run this in your console to install Thunder from the command line:
+```
+composer create-project thunder/thunder-project thunder --no-interaction --no-install
+cd thunder
+composer install
+composer drupal:scaffold
+cd docroot
+php core/scripts/drupal quick-start thunder
```

Comment:
We should stress, that this is for local testing / development only. Can we give a link to a good installation on production systems?